code/modules/clothing/under/chameleon.dm

```diff
@@ -81,3 +77,26 @@
 	for(var/U in typesof(/obj/item/clothing/under)-blocked)
 		var/obj/item/clothing/under/V = new U
 		src.clothing_choices += V
+
+/obj/item/clothing/under/chameleon/cold
+	heat_conductivity = 1000
+	var/registered_user = null
+
+/obj/item/clothing/under/chameleon/cold/attack_self(mob/user as mob)
+	if(!registered_user || registered_user == user)
+		if(!registered_user)
+			to_chat(usr, "You are registered as the user of this suit")
+			registered_user = user
+		if(!(/obj/item/clothing/under/chameleon/proc/Change_Color in verbs))
+			verbs |= /obj/item/clothing/under/chameleon/proc/Change_Color
+			return
+		if(/obj/item/clothing/under/chameleon/proc/Change_Color in verbs)
+			verbs &= ~/obj/item/clothing/under/chameleon/proc/Change_Color
```

Comment:
change this to `verbs -=` and remove the `~`

Reply:
Kind of surprised this one compiles

Reply:
Had to test it myself, `~path` is equal to `0xFFFFFF`